build.sh

```diff
@@ -142,6 +142,10 @@ apt-get -q -y --purge install language-pack-gnome-en
 apt-get -q -y --purge install ibus ibus-clutter ibus-gtk ibus-gtk3 ibus-qt4
 apt-get -q -y --purge install ibus-unikey ibus-anthy ibus-pinyin ibus-m17n
 apt-get -q -y --purge install im-switch
+
+#Debuilding the metapackages
+./script.sh
```

Comment:
https://travis-ci.org/fossasia/meilix/builds/256556730?utm_source=github_status&utm_medium=notification#L9706

make sure it is also copied to chroot